influxdb_connector/influxdb_write_tester.py

```diff
@@ -2,29 +2,53 @@
 import argparse
 import pandas as pd
 from collections import defaultdict
+
 # from cnvrgv2 import Cnvrg
 import influxdb_client, os, time
 from influxdb_client import InfluxDBClient, Point, WritePrecision
 from influxdb_client.client.write_api import SYNCHRONOUS
 
 cnvrg_workdir = os.environ.get("CNVRG_WORKDIR", "/cnvrg")
 url = "placeholder"
-org =  "placeholder"
+org = "placeholder"
```

Comment:
Should read these value by some mean before calling 

``` python: 
    # writing data to influxdb api
   client = influxdb_client.InfluxDBClient(url=url, token=token, org=org)
```

Reply:
Now reading from txt file.

Reply:
influxdb_write_tester.py replaced with influxdb_write_tester.ipynb to support direct csv writing and better interactivity